.babelrc

```diff
@@ -1,5 +1,5 @@
 {
-  "stage": 1,
+  "stage": 0,
```

Comment:
This is weird but the Babel class property bind thing is faster than `func.bind`, so why not? I'm going through all the components anyway argh